gnucash/report/trep-engine.scm

```diff
@@ -2069,6 +2069,12 @@ warning will be removed in GnuCash 5.0"))
        (else
         (string-contains str transaction-matcher))))
 
+    (define (locale-string<? a b)
+      (< (gnc-collation-compare a b) 0))
+
+    (define (locale-string>? a b)
+      (> (gnc-collation-compare a b) 0))
+
```

Comment:
Do this in core-utils.scm so it's available to all scheme code instead of just trep-engine.scm users.